docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,177 @@
 
-    1. Do the changelogs match expectations from `git log`?
-    1. Have all the version strings been updated?
-    1. Does the diff from `origin/main` (or `origin/train-xxx` if it’s a point release) look correct?
+    1. Does `git log` look as expected, showing all commits between the previous tag, and the new tag?
+    1. When running a `git diff` from `origin/main`, does the diff look correct, including:
+        - Do the changelogs look as expected with a version bump and commit message summaries?
+        - Have all the version strings in `package.json` been updated?
+        - Do the files changed look as expected for the major release?
 
-1. The release script will print some commands to run to push the train branch to the server.  **It's best to copy and paste these so you don't mix them up.**
+1. The release script will print some commands to run to push the train branch and the tag to the repo. You might consider doing all other commands in a new terminal tab to not lose the print out. **It's best to copy and paste the push commands so you don't mix them up.** The deploy to staging will begin once `deploy_and_tag` finishes in CI which runs after all tests have passed.
 
-1. The release script will also print a URL which you can use to open a PR to merge the train branch back into the main branch
+1. The release script will also print a URL which you can use to open a PR to merge the train branch back into the main branch.
 
-1. Finally, the release script will print out a bug template.  Copy that template and open a deployment bug in bugzilla under `Cloud Services :: Operations: Deployment Requests` ([example][example-deployment-bug]) even if the component says it's deprecated. Remember to include:
+1. The release script will print out a bug template.  Copy that template and open a deployment bug in bugzilla under `Cloud Services :: Operations: Deployment Requests` ([example][example-deployment-bug]) even if the component says it's deprecated. Remember to include:
 
-    1. Notes from the deploy doc, particularly any server side changes that need to happen as part of this deployment.
-    1. Links to the needs:qa label on GitHub.
+    1. Any notes that were added during the sprint from the [deploy doc][deployment-doc] for the train, particularly any server side changes that need to happen as part of this deployment.
+    1. Links to the `needs:qa` label on GitHub.
     1. Links to the release tag on GitHub.
     1. Links to pertinent changelogs.
 
-#### Operations staff will take it from there…
-1. Ensure that any configuration changes noted in the deployment bug land in [cloudops-deployment][cloudops-deployment].
+1. Finally, when your train PR into `main` is approved and the deploy to stage has finished, merge the PR. If CI fails and it appears to be a flaky test, rerun the test from failed. If it continues failing, let the team know. There may be a problem that warrants a dot release.
```

Comment:
Or if the fix is non-trivial, we may need to roll the release back with SRE's help?